packages/gatsby-theme-docs/src/components/cards.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { designSystem, ContentNotifications } from '@commercetools-docs/ui-kit';
+
+// Explanation about the following sizes:
+// https://github.com/commercetools/commercetools-docs-kit/pull/427#discussion_r425442556
+const cardNarrowMinWidth = '242px';
+const cardRegularMinWidth = '328px';
+const CardsContainer = styled.div`
+  display: grid;
+  gap: ${designSystem.dimensions.spacings.m};
+  grid-auto-columns: 1fr;
+  grid-template-columns: ${(props) =>
+    `repeat(auto-fill, minmax(${
+      props.narrow ? cardNarrowMinWidth : cardRegularMinWidth
+    }, 1fr))`};
+`;
+
+const Cards = (props) => {
+  try {
+    return (
+      <CardsContainer {...props}>
+        {React.Children.map(props.children, (child) => {
+          if (!child.props || child.props.mdxType !== 'Card') {
+            throw new Error(
+              `Children of <Cards> must be a <Card> component and not "${
+                child.props ? child.props.mdxType : child
+              }"`
+            );
+          }
+          return React.cloneElement(child, [props], [...child.props.children]);
```

Comment:
Why is `[props]` an array?

Reply:
I'm not sure, I just used the code sample as is:

```
React.cloneElement(
  element,
  [props],
  [...children]
)
```

https://reactjs.org/docs/react-api.html#cloneelement

Reply:
🤦‍♂️

Sorry but you can't just copy paste stuff like this without understanding what it does.
The docs example indicates that the arguments are optional, with the `[ ]` notation.

In this case you do it like this:

```js
React.cloneElement(element, props, children)
```

It would be better though to be explicit which props the parent should inject to the children.

```js
React.cloneElement(element, { foo: 'bar'}, children)
```

Which props does the `<Card>` component need from the parent?